src/OhmmsData/libxmldefs.h

```diff
@@ -70,7 +70,8 @@ bool putContent(T& a, xmlNodePtr cur)
   std::istringstream
   stream((const char*)(xmlNodeListGetString(cur->doc, cur->xmlChildrenNode, 1)));
   stream >> a;
-  return stream.good();
+  return !stream.fail();
```

Comment:
Could you revert these changes or there is a reason? It seems irrelevant to this PR.

Reply:
The put function logic didn't consistently work without this, since stream.good() returns false if the stream reached eof.

Reply:
Changing this may affect all other places using this function.
If xml input doesn't provide sufficient values to feed the request, it is a failure.
Please give your logic with an example.

Reply:
!stream.fail() is correct. Please remove all the commented stream.good().

Reply:
I did it.